README.md

```diff
@@ -0,0 +1,205 @@
+
+> Here you can visit my live demo :
+
+- # [Live Demo](https://ellyboi.github.io/Concert-page/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 💻 Summer concert 2023 Presentation <a name="vgs-presentation"></a>
+
+> Here you can see my presentation :
+
+- [Vido Games Global Summit 2023 on LOOM](https://www.loom.com/share/8ad34db0cfd2470ab56519fdb73029be)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Clone the repository by clicking on the 'Code' button and copy the link
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+npm i
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:Ellyboi/Concert-page.git
+```
+
+-
+
+### Install
+
+Install this project with:
+
+### Usage
+
+To run the project, execute the following command:
+Open the index.html file in a web browser
+
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+To run the webhint linter
+
+```sh
+  npx hint .
+```
+
+To run the stylehint linter
+
+```sh
+  npx stylehint "**/*.{css,scss}"
+```
+
+To run the ESLint linter
+
+```sh
+  npx eslint .
+```
+
+### Deployment
+
+You can deploy this project using:
+-Go to the main folder of the project and double-click on the index.html file.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Vitoesi Elijah** 
+
+- GitHub: [@Ellyboi](https://github.com/Ellyboi)
+- Twitter: [@ElijahVitoesi](https://twitter.com/ElijahVitoesi)
+- LinkedIn: [Vitoesi Elijah](https://www.linkedin.com/in/vitoesi-elijah-61961611a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Buy Tickets
+- Schedule
+- Locate the place on map
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project star it!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Cindy Shin<br>
+Here is the Original design idea by Cindy Shin in Behance 👉[Template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [licensed](./LICENCE) .
```

Comment:
[Required] I've noticed that the `License link` on your `readme file` does not lead to the License in the project root directory, please consider changing the License Link to the one below to fix this. A working License link informs someone who would like to use your code or project how they are Licensed. You mis spelled the name of the License file.

```
This project is [MIT](./LICENSE) licensed.
```
![Screenshot from 2023-03-17 18-20-45](https://user-images.githubusercontent.com/16392046/225947259-687ef763-81d1-4031-a2b9-c1cd17c019d1.png)